modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TableViewSkinTest.java

```diff
@@ -46,19 +66,138 @@ public void test_JDK_8188164() {
         Scene scene = new Scene(tableView);
         scene.getStylesheets().add(TableViewSkinTest.class.getResource("TableViewSkinTest.css").toExternalForm());
 
-        Toolkit tk = Toolkit.getToolkit();
+        stageLoader = new StageLoader(scene);
```

Comment:
Note: I just cleaned up this test to also use the more safe `stageLoader` approach.
Unfortunately the diff looks a bit weird...